server/src/main/java/io/spine/server/commandbus/CommandBus.java

```diff
@@ -228,7 +228,7 @@ protected void dispatch(CommandEnvelope command) {
      * @return a set of classes of supported commands
      */
     @Internal
-    public final Set<CommandClass> registeredCommandClasses() {
+    public Set<CommandClass> registeredCommandClasses() {
```

Comment:
Please review this change. I might be wrong, but this method still looks `final` by its nature. As it was prior to this changeset.

Reply:
The `CommandBus` is a final class. The unnecessary modifier was hinted by IDEA.